Sources/App/Models/Speaker/Speaker.swift

```diff
@@ -46,9 +46,7 @@ extension Speaker: JSONRepresentable {
   
   func makeJSON() throws -> JSON {
     guard let userJSON = try user()?.makeJSON() else {
-      var json = JSON()
-      try json.set(Keys.id, id)
-      return json
+      throw Abort(.internalServerError, reason: "Speaker is no have User")
```

Comment:
`Speaker doesn't have associated User.` for reason.

Reply:
sure, thank you. Fixed.